includes/Story_Post_Type.php

```diff
@@ -345,9 +345,6 @@ public function clear_user_posts_count( $post_id, $post ) {
 	 * @return bool|string Whether the post type should have an archive, or archive slug.
 	 */
 	public function get_has_archive() {
-		if ( ! $this->experiments->is_experiment_enabled( 'archivePageCustomization' ) ) {
```

Comment:
You'd also want to remove the `$experiments` class property on line 79 as well as the updating the constructor accordingly.

Reply:
Still need to update the constructor:

![Screenshot 2021-12-06 at 10 29 28](https://user-images.githubusercontent.com/841956/144821534-66d35aca-3083-4ce2-9910-97fd0066f510.png)

I also just saw that the doc block for the constructor is incorrect.

We'd want to change it like so:

```php
/**
 * Story post type constructor.
 *
 * @since 1.12.0
 *
 * @param Settings    $settings     Settings instance.
 *
 * @return void
*/
public function __construct( Settings $settings ) {
	$this->settings    = $settings;
}
```